core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -479,25 +479,33 @@ class LogManager(logDirs: Seq[File],
 
     try {
       for ((dir, dirJobs) <- jobs) {
-        dirJobs.foreach(_.get)
+        val hasErrors = dirJobs.exists {
+          future =>
+            try {
```

Comment:
You can use `scala.util.Try` to wrap the call and get a `Success` or `Failure`.

Reply:
Good idea, done.